lib/util/api.ts

```diff
@@ -71,3 +72,17 @@ export function removeUnusedQueryParams(params: Record<string, any>): void {
   delete params.maxWalkDistance
   delete params.maxBikeDistance
 }
+
+/** Gets the default URL params when reinitializing search */
+export function getDefaultQuery() {
+  return {
+    date: getCurrentDate(),
+    departArrive: 'NOW',
+    intermediatePlaces: [], // required to avoid crash
+    mode: 'WALK,TRANSIT', // obsolete but required to avoid crash
+    numItineraries: 3,
```

Comment:
Apparently it is, see https://github.com/opentripplanner/otp-react-redux/pull/1232#issuecomment-2165761439

Reply:
I added a comment for this. Currently, especially for transit, we return 3 itineraries. If `numItineraries` is not set, it would return 7 itineraries.